authors/apps/articles/models.py

```diff
@@ -160,3 +160,13 @@ class Favourites(models.Model):
         Article, related_name="article_id",
         on_delete=models.CASCADE, null=True)
     favourite = models.BooleanField(default=False)
+
+
+class ReportArticle(models.Model):
+    """
+    Model for reporting article
+    """
+    article = models.ForeignKey(Article, on_delete=models.CASCADE)
+    reported_by = models.ForeignKey(User, on_delete=models.CASCADE)
+    reason = models.CharField(max_length=200)
```

Comment:
@VivianDoreen I think a max length of 200 for reporting an article is too short.
I would consider using a TextField().
What do you think?

Reply:
I think its ok for the max_length to be 200 because in most cases the reasons for reporting are usually brief. or what do you think would be the appropriate max_length

Reply:
@VivianDoreen, 200 characters is just about 40 words, that may not be enough for me to mention exactly what is wrong with the article I am reporting.
Therefore I textfield would be ideal since it doesn't limit size of the articles.

Reply:
Ok, @Opio-Emmanuel-Omona